src/Atmos/Model/AtmosModel.jl

```diff
@@ -52,6 +54,78 @@ struct AtmosModel{O,RS,T,M,P,R,SU,S,BC,IS} <: BalanceLaw
   init_state::IS
 end
 
+abstract type AtmosConfiguration end
+struct AtmosLESConfiguration <: AtmosConfiguration end
+struct AtmosGCMConfiguration <: AtmosConfiguration end
```

Comment:
Unfortunately, we should probably reconsider these names because #689 will rename `LES_Configuration()` to `Atmos_LES_Configuration()`, etc. Maybe `AtmosLESConfigType`?

Reply:
I'm fine with `AtmosLESConfigType`.